gammapy/spectrum/models.py

```diff
@@ -1040,6 +1041,75 @@ def evaluate(energy, amplitude, reference, ecut, index_1, index_2):
         return pwl * cutoff
 
 
+class PLSuperExpCutoff4FGL(SpectralModel):
+    r"""Spectral super exponential cutoff power-law model used for 4FGL.
+
+    This model parametrisation is very similar, but slightly different from
+    `PLSuperExpCutoff3FGL` or `ExponentialCutoffPowerLaw3FGL`
+
+    .. math::
+        \phi(E) = \phi_0 \cdot \left(\frac{E}{E_0}\right)^{-\Gamma_1}
+                  \exp \left( a \left(E_0 ^{\Gamma_2} - \E^{\Gamma_2} \right)
```

Comment:
I think here you have to remove the `\` in front of the `E` to fix the formatting?

When I run `make docs-sphinx` and `make docs-show` locally and browse to the API docs of your class, I see this:

![Screenshot 2019-07-22 at 10 00 23](https://user-images.githubusercontent.com/852409/61615962-8d13ba00-ac67-11e9-9e68-ae430e5fe9e6.png)

Reply:
Done!